README.md

```diff
@@ -33,6 +33,14 @@ Omnigres makes Postgres a developer-first application platform. You can deploy a
 The fastest way to try Omnigres out is by using its [container image](https://github.com/omnigres/omnigres/pkgs/container/omnigres):
 
 ```shell
+# First time only:
```

Comment:
Perhaps it would rather make more sense to point to an article (or articles) on how to install these – if none are good, a separate markdown file in our repo, then?